docker/prepare-environment.sh

```diff
@@ -15,3 +19,5 @@ fi
 
 # Enable ding PHP configuration.
 phpenmod ding
+
+mkdir -p /root/.ssh && touch /root/.ssh/known_hosts && ssh-keyscan -H github.com >> /root/.ssh/known_hosts && chmod 600 /root/.ssh/known_hosts
```

Comment:
`ssh-keyscan` is only an encrypted way of fetching the SSH host keys, not a secure way.

Since you don't know the host keys yet you are only guaranteed an encrypted connection to whoever answers you. Welcome, man in the middle.